### ISSUE TITLE
Prop icing affects rpm but shouldn't

### ISSUE THREAD

**Issue by HHS81** [HHS81/c182s#310]
Found out today that icing on a constant speed prop only affects blade pitch and withvthat thrust. No change in rpm or mp. So wee need someone who knows just to alter thrust in jsbsim.

**Comment by HHS81** [comment]
http://www.pilotfriend.com/safe/safety/icing_conditions.htm

**Comment by HHS81** [comment]
@dany93 Is there a way to dynamically alter the hysteresis-limits?

**Comment by dany93** [comment]
I don't know but, after reading [JSBSim Aerodynamics, Effects](http://wiki.flightgear.org/JSBSim_Aerodynamics#Effects)
```
<hysteresis_limits unit="{RAD | DEG}">
    <min> {number} </min>
    <max> {number} </max>
</hysteresis_limits>
```
A number is expected. Not a property. I don"t know if that can be circumvented.

**Comment by hbeni** [comment]
But shouldnt the blue lever change propeller pitch?

**Comment by dany93** [comment]
@HHS81 wrote 
> Is there a way to dynamically alter the hysteresis-limits?

stall-hyst-norm is just a function whose Y value conditionnaly flips from 0 to 1 at two X values (a and b).
I'm basic in nasal, but I have the impression that it would not be difficult to make such a function in nasal. Which could include the dynamically variable a and b parameters.
Or in xml (I greatly prefer the xml code, more robust).

**Comment by HHS81** [comment]
@dany93 yep, I thought that myself tonight. I will do it in xml, is much easier...😀

**Comment by HHS81** [comment]
@hbeni: no, at least not direct since it is a constant speed prop. A govenor keeps the rpm by varying the blade pitch. By pulling the blue lever you simply adjust the rpm, and the govenor makes sure  that the needed blade pitch is hold. JSBSim simulates this very well per default, but you can create your own govenor of course in JSBSim like the Seneca II does.

**Comment by HHS81** [comment]
@wlbragg there is no carb heat, as the c182s and all other single piston engine cessnas built after 1996 uses fuel-injected engines. They don't need carb... 

When icing occurs you can't do much: trying to get above the clouds where is no humidity and ice will sublimate. Or get down where it is warmer. What I also want to simulate now is the much higher stall speed with ice. That's why I asked about the hysteresis limits....

**Comment by wlbragg** [comment]
Ah, thanks!

**Comment by HHS81** [comment]
Did some tests in X-Plane 11.30 with their Columbia 400. With iced props blade angles reduce, prop thrust reduce, but no reduce of prop speed, no reduce of engine rpm, no reduce of engine power... 😞

**Comment by HHS81** [comment]
Update: it can`t be fixed, since it is a missing feature of jsbsim.
I created a feature-request for jsbsim. We have to wait until someone picks it up.

**Comment by hbeni** [comment]
Just to note and for the archives; its this one: https://github.com/JSBSim-Team/jsbsim/issues/138
